ckanext/georchestra/plugin.py

```diff
@@ -202,10 +202,6 @@ def configure(self, main_config):
                 main_config[i] = schema[i]['default']
         if len(errors):
             raise ConfigError("\n".join(errors))
-        # make sure all the strings in the config are unicode formatted
-        for key, value in main_config.iteritems():
-            if isinstance(value, str):
-                main_config[key] = six.text_type(value, encoding='utf-8')
```

Comment:
Then `six` is probably useless to import now.

Reply:
True, but not for long. It comes back in next fix...